Gemfile.lock

```diff
@@ -52,6 +52,3 @@ DEPENDENCIES
   cucumber (~> 1.3.9)
   ronn
   sinatra
-
-BUNDLED WITH
```

Comment:
I like to commit with `git commit -vp`. It first asks you for each hunk do you want to stage it (like `git add -p`), and then in renders the full diff of the commit when you're writing the commit message in the text editor. That way you review all that you stage into a commit twice.